### PR TITLE
Added client-registration-policy/providers endpoint

### PR DESCRIPTION
**Pull request by agagancarczyk**
https://www.keycloak.org/docs-api/15.0/rest-api/index.html#_client_registration_policy_resource

### CODE REVIEW

File: src/client.ts [hunk 1]
```diff
@@ -41,6 +42,7 @@ export class KeycloakAdminClient {
   public clientScopes: ClientScopes;
   public clientPolicies: ClientPolicies;
   public identityProviders: IdentityProviders;
+	public clientRegistrationPolicies : ClientRegistrationPolicies;
```
Comment (jonkoops) [inline src/client.ts:45]: The indentation here seems off, make sure it's aligned with the rest here.
```suggestion
  public clientRegistrationPolicies: ClientRegistrationPolicies;
```

File: src/client.ts [hunk 2]
```diff
@@ -77,6 +79,7 @@ export class KeycloakAdminClient {
     this.clientScopes = new ClientScopes(this);
     this.clientPolicies = new ClientPolicies(this);
     this.identityProviders = new IdentityProviders(this);
+		this.clientRegistrationPolicies = new ClientRegistrationPolicies(this);
```
Comment (jonkoops) [inline src/client.ts:82]: Same here, make sure to indent with spaces.
```suggestion
    this.clientRegistrationPolicies = new ClientRegistrationPolicies(this);
```

File: test/clientRegistrationPolicies.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
+// tslint:disable:no-unused-expression
```
Comment (jonkoops) [inline test/clientRegistrationPolicies.ts:1]: Which expression here is unused?

**Review by jonkoops — CHANGES_REQUESTED** [review]

File: src/client.ts (hunk 1 above)
Comment (agagancarczyk) [reply src/client.ts:45]: thanks, yeah, something strange is happening. I'll fix it.

File: package-lock.json [hunk 4]
```diff
@@ -1,12 +1,12 @@
 {
   "name": "@keycloak/keycloak-admin-client",
-  "version": "19.0.0-dev.43",
+  "version": "19.0.0-dev.44",
```
Comment (jonkoops) [inline package-lock.json:3]: This file needs to be removed.

**Review by jonkoops — CHANGES_REQUESTED** [review]

**Review by jonkoops — APPROVED** [review]

**Pull request merged**